cli/azd/pkg/rzip/rzip.go

```diff
@@ -12,47 +12,77 @@ import (
 	"strings"
 )
 
+// CreateFromDirectory compresses a directory into a zip file.
 func CreateFromDirectory(source string, buf *os.File) error {
 	w := zip.NewWriter(buf)
+	defer w.Close()
+
+	// Walk through the source directory
 	err := filepath.WalkDir(source, func(path string, info fs.DirEntry, err error) error {
 		if err != nil {
 			return err
 		}
 
-		if info.IsDir() {
-			return nil
-		}
-		fileInfo, err := info.Info()
+		// Fetch file info (Lstat avoids following symlinks)
+		fileInfo, err := os.Lstat(path)
 		if err != nil {
 			return err
 		}
 
-		header := &zip.FileHeader{
-			Name: strings.Replace(
-				strings.TrimPrefix(
-					strings.TrimPrefix(path, source),
-					string(filepath.Separator)), "\\", "/", -1),
-			Modified: fileInfo.ModTime(),
-			Method:   zip.Deflate,
+		// Skip symbolic links
+		if fileInfo.Mode()&os.ModeSymlink != 0 {
```

Comment:
I'm actually thinking that maybe we want to keep venvs excluded by default.  I don't think we have a good use case for including venvs in a zip folder.

@pamelafox - Can you confirm this?  And also do you think that we should include __pycache__ by default if they have a .ignore file?  I'm thinking it will be large and might not be needed in any scenario.